en/intro/conventions.rst

```diff
@@ -35,10 +44,10 @@ When you create links using ``this->Html->link()``, you can use the following
 conventions for the url array::
 
     $this->Html->link('link-title', [
-        'prefix' => 'MyPrefix' // PascalCased
-        'plugin' => 'MyPlugin', // PascalCased
-        'controller' => 'ControllerName', // PascalCased
-        'action' => 'actionName' // camelBacked
+        'prefix' => 'MyPrefix' // PascalCase
+        'plugin' => 'MyPlugin', // PascalCase
+        'controller' => 'ControllerName', // PascalCase
+        'action' => 'actionName' // camelCase
```

Comment:
This is not camelCase, it's camelBacked. CamelCase is same as PascalCase

Reply:
I would also prefer if we used CamelCase since that is everywhere else as such.

Reply:
> This is not camelCase, it's camelBacked. CamelCase is same as PascalCase

https://medium.com/better-programming/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841


and other:

![image](https://user-images.githubusercontent.com/50879411/71027754-a3288800-20ea-11ea-8b9a-a7bf16d123b4.png)


"CamelBacked" is variation and synonym.    https://en.wikipedia.org/wiki/Camel_case


thanks you